frontend/src/js/components/editors/template/TemplateLabelSettings.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+
+var TemplateLabelSettings = React.createClass({
+    propTypes: {
+        label: React.PropTypes.object,
+        update: React.PropTypes.func
+    },
+    updateLabelValue(key) {
+        const self = this;
+        let validator = parseFloat;
```

Comment:
Can we also validate that it's between 0 and 1 if it's a float? Anything greater than 1 is invalid and will make things complain.